db/schema.rb

```diff
@@ -10,34 +10,33 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 202202071123100) do
-
+ActiveRecord::Schema[7.0].define(version: 202202071123100) do
```

Comment:
Can you revert these schema changes since there's no migration here?

Reply:
yea I think these migration changes came in when I rebased main.

Wasn't sure if I should commit them or not I can undo

Reply:
General rule of thumb is commit them if you've intended to change the schema (by writing a migration), but not if it's unintended (for example because your local database was in a different state etc).